controllers/installation_controller.go

```diff
@@ -436,3 +445,15 @@ func (r *InstallationReconciler) applyDeletionPolicy(ctx context.Context, log lo
 	inst.SetAnnotations(annotations)
 	return r.Update(ctx, inst)
 }
+
+func createPorterGRPCClient(ctx context.Context) (porterv1alpha1.PorterClient, error) {
+	conn, err := grpc.DialContext(ctx, "porter-grpc-service:3001", grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
I don't like that this is hardcoded and trying to resolve what this would be beyond the dns entry in the namespace it lives in.  We can deal with this for now, but this should be configurable.

Reply:
Maybe get the service of the porter-grpc-service (deployment should name this) and then use the service name/port? I don't want to make the assumption that this is what we are naming this.

Reply:
I agree, this is not great. Can you at least put a TODO in here to come back and clean this up? I can cut an issue for it too so we don't lose it. Honestly I feel like this is moving more and more towards having the controller actually manage the grpc server itself then it will know the context, it will be able to properly handle config for the server, and optionally start it based on a flag.

Reply:
Added TODO, resolving.